packages/editor/src/lib/exports/fetchCache.ts

```diff
@@ -0,0 +1,27 @@
+import { FileHelpers, assert, fetch } from '@tldraw/utils'
+
+export function fetchCache<T>(cb: (response: Response) => Promise<T>, init?: RequestInit) {
+	const cache = new Map<string, Promise<T | null>>()
```

Comment:
let's add a TODO here that we should consider making sure this doesn't grow unbounded in the future

Reply:
good call